explorer/filters.go

```diff
@@ -5,6 +5,7 @@ import (
 	"sort"
 	"strconv"
 
+	"github.com/pkg/errors"
```

Comment:
should probably just the errors package

Reply:
great catch, no idea why it didn't default to it...